tests/conftest.py

```diff
@@ -1,38 +1,5 @@
-# test environment
-# Copyright (C) 2020  Nguyễn Gia Phong
-#
-# This file is part of palace.
-#
-# palace is free software: you can redistribute it and/or modify it
-# under the terms of the GNU Lesser General Public License as published
-# by the Free Software Foundation, either version 3 of the License,
-# or (at your option) any later version.
-#
-# palace is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public License
-# along with palace.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
Please give this one a header (with your name of course, I haven't touched this).